src/commands/ai/models/info.ts

```diff
@@ -16,48 +19,95 @@ export default class Info extends Command {
   }
 
   static args = {
-    model_resource: Args.string({description: 'resource ID or alias of the model resource', required: true}),
+    model_resource: Args.string({description: 'resource ID or alias of the model resource'}),
   }
 
   public async run(): Promise<any> {
     const {args, flags} = await this.parse(Info)
     const {app} = flags
     const {model_resource: modelResource} = args
+    const synthesizedModels: Array<ModelResource> = []
+    let listOfProvisionedModels: Array<ModelResource>  = []
 
     const modelInfo = async () => {
-      return this.herokuAI.get<ModelResource>(`/models/${this.apiModelId}`, {
+      const modelInfoResponse = await this.herokuAI.get<ModelResource>(`/models/${this.apiModelId}`, {
         headers: {authorization: `Bearer ${this.apiKey}`},
       })
         .catch(error => {
           if (error.statusCode === 404) {
-            ux.warn(`We can’t find a model resource called ${color.addon(modelResource)}.\nRun ${color.cmd('heroku addons -a <app>')} to see a list of model resources.`)
+            ux.warn(`We can’t find a model resource called ${color.yellow(modelResource)}.\nRun ${color.cmd('heroku ai:models:info -a <app>')} to see a list of model resources.`)
```

Comment:
```suggestion
            ux.warn(`We can’t find a model resource called ${color.addon(modelResource)}.\nRun ${color.cmd('heroku ai:models:info -a <app>')} to see a list of model resources.`)
```

`color.yellow` is an old and unsupported way of getting the text to appear yellow ([reference](https://github.com/heroku/heroku-cli-color)).